src/lib.rs

```diff
@@ -227,11 +227,13 @@ impl BootInformation {
         unsafe { &*self.inner }
     }
 
-    fn get_tag<'a>(&'a self, typ: u32) -> Option<&'a Tag> {
+    /// Search for a tag by `type` identifier.
+    pub fn get_tag<'a>(&'a self, typ: u32) -> Option<&'a Tag> {
```

Comment:
We don't use `get_*` for anything else - I think we should use this as an opportunity to just rename this to `tag`.